cmd/oras/root/cp.go

```diff
@@ -179,24 +180,11 @@ func doCopy(ctx context.Context, printer *output.Printer, src oras.ReadOnlyGraph
 	}
 	if opts.TTY == nil {
 		// none TTY output
-		extendedCopyOptions.OnCopySkipped = func(ctx context.Context, desc ocispec.Descriptor) error {
-			committed.Store(desc.Digest.String(), desc.Annotations[ocispec.AnnotationTitle])
-			return printer.PrintStatus(desc, promptExists)
-		}
-		extendedCopyOptions.PreCopy = func(ctx context.Context, desc ocispec.Descriptor) error {
-			return printer.PrintStatus(desc, promptCopying)
-		}
-		extendedCopyOptions.PostCopy = func(ctx context.Context, desc ocispec.Descriptor) error {
-			committed.Store(desc.Digest.String(), desc.Annotations[ocispec.AnnotationTitle])
-			if err := output.PrintSuccessorStatus(ctx, desc, dst, committed, printer.StatusPrinter(promptSkipped)); err != nil {
-				return err
-			}
-			return printer.PrintStatus(desc, promptCopied)
-		}
-		extendedCopyOptions.OnMounted = func(ctx context.Context, desc ocispec.Descriptor) error {
-			committed.Store(desc.Digest.String(), desc.Annotations[ocispec.AnnotationTitle])
-			return printer.PrintStatus(desc, promptMounted)
-		}
+		copyHandler := status.NewTextCopyHandler(printer, dst)
```

Comment:
Should merge initialization to `display.NewCopyHandler` and gets both handler for status and metadata output.

Reply:
Done